google/datalab/kernel/__init__.py

```diff
@@ -34,18 +34,6 @@
 import google.datalab.storage.commands
 import google.datalab.utils.commands
 
-# mlalpha modules require TensorFlow, CloudML SDK, and DataFlow (installed with CloudML SDK).
-# These are big dependencies and users who want to use Bigquery/Storage features may not
-# want to install them.
-# This __init__.py file is called when Jupyter/Datalab loads magics on startup. We don't want
-# Jupyter+pydatalab fail to start because of missing TensorFlow/DataFlow. So we ignore import
-# errors on mlalpha commands.
-try:
-  import datalab.mlalpha.commands
-except:
-  print('TensorFlow and CloudML SDK are required.')
```

Comment:
Without tensorflow, if you import google.datalab.ml the error message you get should be clear enough.